Client/Frontend/AuthenticationManager/AuthenticationSettingsViewController.swift

```diff
@@ -157,7 +157,7 @@ class TouchIDSetting: Setting {
         let accessoryContainer = UIView(frame: control.frame)
         accessoryContainer.addSubview(control)
 
-        let gesture = UITapGestureRecognizer(target: self, action: "switchTapped")
```

Comment:
Why not move to the cleaner syntax mentioned last Thursday [1]? Its as good of a time as any since we need to change all of them anyways.

[1] https://medium.com/swift-programming/swift-selector-syntax-sugar-81c8a8b10df3#.k4aawoosx